internal/backend/init/init.go

```diff
@@ -101,6 +105,10 @@ func Init(services *disco.Disco) {
 			)
 		},
 	}
+
+	RemovedBackends = map[string]bool{
+		"etcd": true,
```

Comment:
This seems like a nice way to do it. One small addition I might suggest is to make this a map from string to string where the values are the version number where we removed the backend, so that the error message could say `"The %q backend isn't supported from Terraform %s onwards."` and thus give folks a hint about which is the latest version they can use until they are ready to migrate away from the unsupported backend.

Or alternatively, I guess it could just be a map from backend name to full error message and the logic that returns it could just return the message verbatim, and then we could just set the string here to literally `"The \"etcd\" backend isn't supported in Terraform v0.13 and later."` (or similar) and keep the inline logic that deals with this in the three separate places reduced to just echoing out a hard-coded error message. :thinking: